src/main/java/ch/njol/skript/util/Timespan.java

```diff
@@ -131,118 +173,112 @@ public static Timespan parse(final String s) {
 				
 			}
 		}
+
 		return new Timespan(t);
 	}
 	
-	private final long millis;
-	
 	public Timespan() {
 		millis = 0;
 	}
 	
-	public Timespan(final long millis) {
+	public Timespan(long millis) {
 		if (millis < 0)
 			throw new IllegalArgumentException("millis must be >= 0");
 		this.millis = millis;
 	}
 	
 	/**
 	 * @deprecated Use fromTicks_i(long ticks) instead. Since this method limits timespan to 50 * Integer.MAX_VALUE.
-	 * @addon I only keep this to allow for older addons to still work. / Mirre
+	 * Keeping this for older addons to stay working.
 	 */
 	@Deprecated
-	public static Timespan fromTicks(final int ticks) {
-		return new Timespan(ticks * 50L);
+	public static Timespan fromTicks(int ticks) {
+		return new Timespan(ticks * Times.TICK.time);
 	}
 	
-	public static Timespan fromTicks_i(final long ticks) {
-		return new Timespan(ticks * 50L);
+	public static Timespan fromTicks_i(long ticks) {
+		return new Timespan(ticks * Times.TICK.time);
 	}
-	
+
 	public long getMilliSeconds() {
 		return millis;
 	}
 	
 	public long getTicks_i() {
-		return Math.round((millis / 50.0));
+		return Math.round((float) (millis / Times.TICK.time));
```

Comment:
Why is this a float and not a double? Float is doable for an integer like what the deprecated methods do, but for long it should be a double. There is a pull request to remove the deprecated methods aswell.

Reply:
The whole rounding is not needed anyways, updated.